docs/admin/deployment/onboarding_admins.rst

```diff
@@ -31,60 +31,49 @@ To set up AW2, follow these steps:
+5. Launch the Tails Cloner (**Applications ▸ Tails ▸ Tails Cloner**).
+   Select the option to **Clone the current Tails.** This will delete all data on the AW2 USB drive.
+6. Check the box marked **Clone the current Persistent Storage.**
+7. Click **Install**.
+8. Choose a unique passphrase for AW2 and record it securely.
+9. Shut down AW1.
+10. Boot AW2, set an Admin password on the welcome screen, unlock the Persistent Storage, and
+    enable all the options in the Persistent Storage settings.
+11. Record the new passphrase for AW2 securely.
 12. Generate a new keypair on AW2 using the following command:
 
     ``ssh-keygen -t rsa -b 4096``
 
     When prompted, store the keypair in the default location.
-13. Shut down AW2.
-14. Boot AW1, unlock its persistent volume, and set an admin password on
-    the welcome screen.
-15. Open the file manager (**Applications ▸ Accessories ▸ Files**).
-16. Insert AW2 and unlock it.
-17. In a terminal, type the following commands to authorize the newly created SSH keypair
-    on your servers:
-
-    ``ssh-copy-id -i /media/amnesia/TailsData/openssh-client/id_rsa.pub app``
-    ``ssh-copy-id -i /media/amnesia/TailsData/openssh-client/id_rsa.pub mon``
-18. Log into the *Journalist Interface* using your admin credentials, and create
-    a new user account with admin rights. Record its passphrase securely;
-    you will add it to the password manager on AW2.
-
-    (You will need to on-board the new admin's 2FA device to complete this step.
-    If this is not possible yet, you can defer it until later.)
-19. Shut down AW1.
-20. Boot AW2, unlock its persistent volume, and set an admin password
-    on the welcome screen.
-21. Boot into AW2 and run the command ``./securedrop-admin tailsconfig`` in
-    ``~/Persistent/securedrop``.
+13. Run the command ``./securedrop-admin tailsconfig`` in ``~/Persistent/securedrop``.
 
     This will set up desktop shortcuts and SSH access.
-22. Confirm that you are able to access ``mon`` and ``app`` via SSH (``ssh app`` and ``ssh mon``).
-23. Confirm that you are able to access the *Source Interface* and the *Journalist
+14. Insert AW1. It should show up in the list of storage devices in the file manager under
+    a label like "7.0 GB Encrypted". Click the label and enter the drive
+    password when prompted to unlock it.
+15. In a terminal, type the following commands to authorize the newly created SSH keypair
+    on your servers:
+    ``ssh-add``
+    ``ssh-add /media/amnesia/TailsData/openssh-client/id_rsa``
+    ``ssh-copy-id app``
+    ``ssh-copy-id mon``
+    ``ssh-add -D``
+16. Confirm that you are able to access ``mon`` and ``app`` via SSH (``ssh app`` and ``ssh mon``).
+17. Confirm that you are able to access the *Source Interface* and the *Journalist
     Interface* using the desktop shortcuts.
-24. :ref:`Initialize a passphrase database <keepassxc_setup>` on AW2.
+18. :ref:`Initialize a passphrase database <keepassxc_setup>` on AW2.
```

Comment:
Would this database not have been copied over from AW1? Should this say to delete the other admin's credentials and add the new one's?

Reply:
That's a good catch, you're exactly right.